lib/activerecord-import/import/result.rb

```diff
@@ -0,0 +1,37 @@
+module ActiveRecord::Import #:nodoc:
+  class Result
+    attr_accessor :failed_instances
+    attr_writer :num_inserts, :ids, :results
+    def initialize(failed_instances, num_inserts, ids, results)
+      @failed_instances = failed_instances
+      @num_inserts = num_inserts
+      @ids = ids
+      @results = results
+    end
+
+    def num_inserts(with_warn: true)
+      warn 'Info: ActiveRecord::Import::Result#num_inserts is not an increased record count. It is the number of inserts it took to import the data.' if with_warn
```

Comment:
This mistake is easy to happen.In fact our other team member made the same mistake.

Reply:
I don't think I like this warning here. I know it's confusing, but this seems to get in the way a bit.

Reply:
@jkowens 
Thanks for review.
OK. I'll think about other approach.

Reply:
@tashirosota I'm good with the warning for `ids` and `results` though. If you just want to remove the warning for `num_inserts` I'm ok with moving forward with this.